components/design/DesignCases/Case/Case.jsx

```diff
@@ -39,12 +39,16 @@ const Case = ({
   const side = (index + 1) % 2 ? 'left' : 'right'
 
   return (
-    <div className={cn(className, id(translations), size, side, { is_touched: isTouched })}>
+    <div
+      className={cn(className, id(translations), size, side, { is_touched: isTouched })}
+      data-testid={`DesignCases:block:case.${id(translations)}`}
+    >
       <div
         className="image-wrapper"
         onTouchStart={handleTouchStart}
         onTouchMove={handleTouchMove}
         onTouchEnd={handleTouchEnd}
+        data-testid={`DesignCases:block:case.${id(translations)}.image`}
```

Comment:
Лучше заменить название id:
```suggestion
        data-testid={`DesignCases:block:case.${id(translations)}.imageWrapper`}
```
Одну и туже вещь лучше всегда называть одинаково, что бы проще было находить общий язык и понимать код

Reply:
Названием data-testid занимаются разработчики, по этому я не могу их поправить.
И ПР по ним ранее был от тебя уже апрувнут.